apps/browser/pages/api/supabase/prompts.ts

```diff
@@ -0,0 +1,51 @@
+import { createClient } from "@supabase/supabase-js";
+import { NextApiRequest, NextApiResponse } from "next";
+const supabase = createClient(
+  process.env.SUPABASE_URL as string,
+  process.env.SUPABASE_SERVICE_ROLE_KEY as string
+);
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  switch (req.method) {
+    case "GET":
+      const { data: prompts, error } = await supabase
+        .from("prompts")
+        .select()
+        .eq("user_email", req.query.email)
```

Comment:
This should be done via a auth token instead of email, otherwise anyone can fetch this info for any email.
Might not matter for the MVP though